content/en/dora_metrics/_index.md

```diff
@@ -51,61 +48,33 @@ Defining and tracking DORA metrics can help you identify areas of improvement fo
-
-Use the information on this page to identify improvements or regressions for each metric, visualize changes, and compare trends over time. DORA metrics can be exported to dashboards or notebooks and be alerted on using [metric monitors][3].
-
-The metrics can also be queried with the [Query timeseries points][4] and [Query timeseries data across multiple products][5] API endpoints.
-
-The metrics provided by DORA Metrics are:
+To start configuring data sources to send deployment and incident events to Datadog, see the [Setup documentation][2].
 
-| Metric | Type | Description |
-| :--- | :--- | :--- |
-| `dora.deployments.count` | count | Used for Deployment Frequency.
-| `dora.change_lead_time` | distribution | Contains the age in `seconds` of the git commits at the time of deployment.
-| `dora.incidents_impact` | count | Tracks the services or teams impacted by incidents. Used for Change Failure Rate with the formula `dora.incidents_impact / dora.deployments.count`. A big time rollup of at least 1 week is recommended to account for time difference between deployments and when the impact starts.
-| `dora.time_to_restore` | distribution | Contains the time in `seconds` between the incident's `started_at` and `finished_at`.
+## Analyze DORA Metrics
 
-All the metrics contain the following tags when available:
-- `service`
-- `team`
-- `env`
-- `repository_id`
+Once you've set up the data sources for your deployment and incident events, navigate to [**Software Delivery** > **DORA Metrics**][4] to identify improvements or regressions for each metric, examine deployments by service or environment, and compare trends over time. 
 
-**Note**: The `severity` tag is available for the `dora.incidents_impact` and `dora.time_to_restore` metrics, if provided through the API.
+{{< img src="dora_metrics/home.png" alt="An Overview of your DORA Metrics calculations and insights in the last week" style="width:100%;" >}}
 
-### Deployment and incident events
+You can access and visualize your DORA metrics and filter them by team, service, repository, environment, and time period on the [**DORA Metrics** page][4]. For more information about the metrics calculated by DORA Metrics, see the [Data Collected documentation][3].
 
-DORA Metrics also provides individual `deployment`, `incident`, and `incident_finished` events in [Event Management][6] with `source:software_delivery_insights`.
+## Explore DORA Metrics events in Event Management
 
-The events can be queried and visualized with the [Events Explorer][7].
+DORA Metrics provides individual `deployment`, `incident`, and `incident_finished` events under the `Software Delivery Insights` source in [Event Management][7]. Navigate to [**Service Management** > **Event Management** > **Explorer**][8] and enter `source:software_delivery_insights` in the search query to filter on DORA Metrics events.
```

Comment:
should we have a link here with the query pre-filled? https://app.datadoghq.com/event/explorer?query=source%3Asoftware_delivery_insights%20&cols=&messageDisplay=expanded-lg&options=&refresh_mode=sliding&sort=DESC&from_ts=1714391730343&to_ts=1714392630343&live=true